### PR TITLE
Cleanup makefile

### PR DESCRIPTION
**Pull request by sergio97**
Fix whitespace and add a success message after successful build.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,28 +24,28 @@ LDFLAGS=\
...
 	-s MODULARIZE=1 \
 	$(CPYTHONROOT)/installs/python-$(PYVERSION)/lib/libpython$(PYMINOR).a \
-  $(LZ4LIB) \
-  -s "BINARYEN_METHOD='native-wasm'" \
-  -s TOTAL_MEMORY=1073741824 \
-  -s ALLOW_MEMORY_GROWTH=1 \
+	$(LZ4LIB) \
+	-s "BINARYEN_METHOD='native-wasm'" \
+	-s TOTAL_MEMORY=1073741824 \
+	-s ALLOW_MEMORY_GROWTH=1 \
```
Comment (sergio97) [inline Makefile]: Fix mix of tabs and spaces. Now all tabs :+1:

File: Makefile [hunk 2]
```diff
@@ -64,19 +64,20 @@ all: build/pyodide.asm.js \
...
-  build/packages.json \
-  build/test.html \
-  build/webworker.js \
-  build/webworker_dev.js
+	build/test.data \
+	build/packages.json \
+	build/test.html \
+	build/webworker.js \
+	build/webworker_dev.js
+	echo -e "\nSUCCESS!"
```
Comment (sergio97) [inline Makefile]: image

**Comment by sergio97** [comment]
That test is passing locally for me. Are these tests known to be flaky?

**Comment by mdboom** [comment]
Yes, that's a known-flaky test.  Merging.  Thanks for the contribution!  \o/

**Pull request merged**